about/about-kubernetes-egress.md

```diff
@@ -0,0 +1,111 @@
+- Restricting egress traffic
+- Outgoing NAT behavior
+- Egress gateways
+
+### Restricting egress traffic
+
+It's a common security requirement and best practice to restrict outgoing connections from the cluster. This is normally
+achieved using [Network Policy]({{site.baseurl}}/about/about-network-policy) to define egress rules for each
+microservices, often in conjunction with a [default deny]({{site.baseurl}}/about/about-network-policy#default-deny)
+policy that ensures outgoing connections are denied by default, until a policy is defined to explicitly allow specific
+traffic.
+
+One limitation when using Kubernetes Network Policy to restrict access to specific external resources, that the external
+resources need to be specified as IP addresses (or IP address ranges) within the policy rules. If the IP addresses
+associated with an external resource change, then every policy that referenced those IP addresses needs to updated with
+the new IP addresses. This limitation can be circumvented using Calico [Network
+Sets]({{site.baseurl}}/security/external-ips-policy), or Calico Enterprise's support for [domain
+names]({{site.baseurl}}/security/calico-enterprise/egress-access-controls) in policy rules.
+
+In addition to using network policy, service meshes typically allow you to configure which external services each pod
+can access. In the case of Istio, {{site.prodname}} can be integrated to enforce network policy at the service mesh
+layer, including [L5-7 rules]({{site.baseurl}}/security/http-methods), as another alternative to using IP addresses in rules. To
+learn more about the benefits of this kind of approach, read our [Adopt a zero trust network model for security
+]({{site.baseurl}}/security/adopt-zero-trust) guide.
+
+Note in addition to everything mentioned so far, perimeter firewalls can also be used to restrict outgoing connections,
+for example to only allow connections to particular external IP address ranges, or external services. However, since
+perimeter firewalls typically cannot distinguish individual pods, the rules apply equally to all pods in the cluster.
+This provides some defense in depth, but cannot replace the requirement for network policy.
+
+### NAT outgoing
+
+Network Address Translation ({% include open-new-window.html text='NAT'
+url='https://en.wikipedia.org/wiki/Network_address_translation' %}) is the process of mapping an IP address in a packet
+to a different IP address as the packet passes through the device performing the NAT. Depending on the use case, NAT can
+apply to the source or destination IP address, or to both addresses.
+
+In the context of Kubernetes egress, NAT is used to allow pods to connect to services outside of the cluster if the pods
+have IP addresses that are not routable outside of the cluster (for example, if the pod network is an overlay).
+
+For example, if a pod in an overlay network attempts to connect to an IP address outside of the cluster, then the
```

Comment:
(needs comma or doesn't read well.) 

For example, if a pod in an overlay network**,** attempts to connect to an IP address outside of the cluster,